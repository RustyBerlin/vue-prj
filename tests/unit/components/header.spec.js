import { shallowMount } from '@vue/test-utils';
import Component from '../../../components/Header.vue';

describe('Test', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(Component);
  });

  test('snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('check header', () => {
    expect(wrapper.contains('header')).toBe(true);
  });
});
