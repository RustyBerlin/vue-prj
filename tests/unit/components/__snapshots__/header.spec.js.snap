// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot 1`] = `"<header id=\\"top\\"><hero-stub></hero-stub> <navigation-stub></navigation-stub></header>"`;
